data/helpers/usersDb.js

```diff
@@ -0,0 +1,42 @@
+const db = require('../../dbConfig')
+
+module.exports = {
+  getUsers,
+  getUser,
+  findUser,
+  addUser,
+  updateUser,
+  removeUser
+}
+
+function getUsers() {
+  return db('users')
+}
+function getUser(id) {
+  return db('users')
+    .where({ id })
+    .first()
+}
+function findUser(body) {
+  return db('users')
+    .where({ username: body.username })
```

Comment:
You could alternatively pull the username property off of the body parameter and then use shorthand syntax here (since the key would be the same as the variable holding the value):
```javascript
function findUser({ username }) {
  return db('users')
    .where({ username })
    .first()
}
```